### ISSUE TITLE
"browsers" options deprecated when upgrade autoprefixer version 9.6.0 (latest)

### ISSUE THREAD

**Issue by DemonCloud** [csstools/postcss-preset-env#133]
Upgrade autoprefixer to 9.6.0 latest version

browsers option deprecated

<img width="723" alt="Screen Shot 2019-06-06 at 11 49 22 AM" src="https://user-images.githubusercontent.com/7436361/59005764-50276b80-8851-11e9-8de7-3cc7c86aa2ae.png">

**Comment by adnasa** [comment]
If I understood the implementation correctly, it looks like consumers of `postcss-preset-env` need to specifically pass down `overrideBrowsersList` and have `browsers=undefined` **as an intermediate solution**  until we get further clarification.

We get a warning if we use `browsers` and not `overrideBrowsersList`

reference: https://github.com/postcss/autoprefixer/blob/7980d0ac3ebb0d1adbbe6a75d248ca3702b2d5f3/lib/autoprefixer.js#L78-L93

**Comment by Antonio-Laguna** [comment]
Using this type of configuration:

```
require('postcss-preset-env')({
  autoprefixer: {
    overrideBrowserslist: config.browserlist
  }
})
```

Will work as expected with no change from this package. @jonathantneal would you like a PR to proxy this to the appropriate config or would you just warn this on README?

**Comment by adnasa** [comment]
@Antonio-Laguna Thanks for the response. then my proposed approach is verified 😃

**Comment by jonathantneal** [comment]
This has been fixed in 6.7.0.
